tests/integration/test_install_basic.py

```diff
@@ -379,9 +379,7 @@ def test_system_and_deploy_work(pipenv_instance_private_pypi):
 @pytest.mark.basic
 @pytest.mark.install
 def test_install_creates_pipfile(pipenv_instance_pypi):
-    with pipenv_instance_pypi() as p:
```

Comment:
I had to debug this test, and I figure may as well use this simplification that existed in the fixture to not create the pipfile in the first place.